### PR TITLE
Inform admin when new OAuth2 account user is created

### PR DESCRIPTION
**Pull request by marwanehcine**
On this PR, when a user account with OAuth2 is created, en entry in journal is added and an email is sent to admin user.

### CODE REVIEW

File: console/src/main/webapp/manager/app/assets/lang/fr.json [hunk 1]
```diff
@@ -263,6 +263,7 @@
   "logs.roleremoved"      : "a été enlevé de l'utilisateur",
   "logs.emailrecoverysent": "Email de récupération du mot de passe envoyé",
   "logs.rolecreated"      : "Role créé",
+  "logs.oauth2usercreated": "Utilisateur OAuth2 créé",
```
Comment (fvanderbiest) [inline console/src/main/webapp/manager/app/assets/lang/fr.json:266]: Thanks !
All lang files should be updated by the PR (using google translate if required)

**Comment by fvanderbiest** [comment]
OK, so if I understand correctly, we now log 3 different "events" on user creation: 
* `pendingusercreated` for local user (ldap backend) when "moderatedSignup=true"  
* `usercreated` for local user when "moderatedSignup=false"
* `oauth2usercreated` when user connects with an external identity provider

Is this correct ?

**Comment by marwanehcine** [comment]
> OK, so if I understand correctly, we now log 3 different "events" on user creation:
> 
> * `pendingusercreated` for local user (ldap backend) when "moderatedSignup=true"
> * `usercreated` for local user when "moderatedSignup=false"
> * `oauth2usercreated` when user connects with an external identity provider
> 
> Is this correct ?

Yes, I guess

File: console/src/main/webapp/manager/app/assets/lang/fr.json (hunk 1 above)
Comment (marwanehcine) [reply console/src/main/webapp/manager/app/assets/lang/fr.json:266]: ok, will be done, thanks

**Pull request closed without merging**